foreach2.js

```diff
@@ -12,3 +12,9 @@
 
 // your code here
 
+for(let i = 0; i < foods.length; i++) {
+   console.log(foods[i].name, "is " + foods[i].level + " delicious")
+}
+foods.forEach(function(food) {
+
+})
```

Comment:
same thing here as in `forEach1.js`. You can take the inside of your `for` loop, replace `foods[i]` with `food`, and it would work.

it looks like you understand how `for` loops work, and you understand how to set up your iterator, but you're just 1 tiny step away from understanding how to put them together.